README.md

```diff
@@ -264,7 +275,7 @@ The generated keystore file then needs placed in a folder called `keystore` in a
 ##### Activate
 Run the following command to activate the HTTPS endpoint:
 
-`docker-compose -f docker-compose.yml -f docker-compose.https.yml up`.
+`docker-compose -f docker-compose.yml -f docker-compose.https.yml up
```

Comment:
I am not sure this is still meant to be here?

If it is meant to be still included in the document it is missing a closing:  `
i.e.  `docker-compose -f docker-compose.yml -f docker-compose.https.yml up`

The line  `docker-compose -f docker-compose.yml -f docker-compose.https.yml up`  has been used early at the top this section which makes me wonder if this should still be included.